spp_programs_compliance_criteria/models/g2p_cycle.py

```diff
@@ -36,3 +37,42 @@ def _get_compliance_criteria_domain(self):
             membership = self.cycle_membership_ids if self.cycle_membership_ids else None
             domain.append(manager_ref._prepare_eligible_domain(membership))
         return OR(domain)
+
+    def _compute_all_members_count(self):
+        for rec in self:
+            domain = rec._get_beneficiaries_domain(None)
+            members_count = self.env["g2p.cycle.membership"].search_count(domain)
+            rec.update({"all_members_count": members_count})
+
+    def open_all_members_form(self):
+        self.ensure_one()
+
+        action = {
+            "name": _("Cycle Members"),
+            "type": "ir.actions.act_window",
+            "res_model": "g2p.cycle.membership",
+            "context": {
+                "create": False,
+                "default_cycle_id": self.id,
+            },
+            "view_mode": "list,form",
+            "domain": [("cycle_id", "=", self.id)],
+        }
+        return action
+
+    def open_members_form(self):
+        self.ensure_one()
+
+        action = {
+            "name": _("Cycle Beneficiaries"),
+            "type": "ir.actions.act_window",
+            "res_model": "g2p.cycle.membership",
+            "context": {
+                "create": False,
+                "default_cycle_id": self.id,
+                "search_default_enrolled_state": 1,
+            },
+            "view_mode": "list,form",
+            "domain": [("cycle_id", "=", self.id)],
+        }
+        return action
```

Comment:
@emjay0921 upon checking, the only difference between this and the original function is the key "name", change this function to use super

```suggestion
    def open_members_form(self):
        self.ensure_one()
        action = super().open_members_form()
        action["name"] = _("Cycle Beneficiaries")
        return action
```